# Message format for sending drive commands to rover.

float32 acc   # -100% to 100% acceleration
float32 steer # -45 to 45 degrees
